README.md

```diff
@@ -33,6 +33,13 @@ Gears could be used together or alone.
 - [![Version](https://img.shields.io/maven-central/v/com.redmadrobot.extensions/resources-ktx?style=flat-square&label=resources-ktx)][resources-ktx] - A set of extensions for accessing resources
 - [![Version](https://img.shields.io/maven-central/v/com.redmadrobot.extensions/viewbinding-ktx?style=flat-square&label=viewbinding-ktx)][viewbinding-ktx] - A set of extensions for dealing with ViewBinding
 
+### :mag_right: **[ViewModelEvents](viewmodel-events/)**
+
+- [![Version](https://img.shields.io/maven-central/v/com.redmadrobot.gears/kotlin?style=flat-square)][viewmodel-events-common] - A set of common entities used in ViewModelEvents
```

Comment:
I think this one could be removed, as `viewmodelevents-common` is a technical artefact